Content.Shared/GameObjects/ContentNetIDs.cs

```diff
@@ -95,6 +95,8 @@ public static class ContentNetIDs
         public const uint TAG = 1086;
         // Used for clientside fake prediction of doors.
         public const uint DOOR = 1087;
+        public const uint SPAWN_AFTER_INTERACT = 1088;
+        public const uint DISASSEMBLE_ON_ACTIVATE = 1089;
```

Comment:
You don't send any component messages/states from these components, so these aren't needed.

Reply:
They're needed for subscription to LocalEvents. The EventBus won't register components without a NetID.